gdx/src/com/badlogic/gdx/maps/tiled/TiledMapImageLayer.java

```diff
@@ -27,13 +27,29 @@ public class TiledMapImageLayer extends MapLayer {
 	private float y;
 	private boolean repeatX;
 	private boolean repeatY;
+	private boolean supportsTransparency;
 
 	public TiledMapImageLayer (TextureRegion region, float x, float y, boolean repeatX, boolean repeatY) {
 		this.region = region;
 		this.x = x;
 		this.y = y;
 		this.repeatX = repeatX;
 		this.repeatY = repeatY;
+		this.supportsTransparency = checkTransparencySupport(region);
+	}
+
+	/** TiledMap ImageLayers can support transparency through tint color if the image provided is in PNG or GIF format. Here we
+	 * check to see if the file type supports transparency
+	 *
+	 * @param region TextureRegion of the ImageLayer
+	 * @return boolean */
+	private boolean checkTransparencySupport (TextureRegion region) {
+		String texturePath = region.getTexture().toString().toLowerCase();
+		return texturePath.endsWith(".png") || texturePath.endsWith(".gif");
```

Comment:
I'll take another look, this was a fairly quick change when I learned the differences in in how Tiled handles jpgs vs, png, gif.

I appreciate you taking the time to look this over. I'll look through the responses, give it another go and try to make the libGDX team proud.

Reply:
I wont push any changes until I feel comfortable. But I think I have a MUCH cleaner solution thanks to your comment.
I'd replace the current check in TiledMapImageLayer with something like this
```
/** TiledMap ImageLayers can support transparency through tint color if the image provided
	 *  supports the proper pixel format.
	 *  Here we check to see if the file supports transparency by checking the format of the TextureData.
	 *
	 * @param region TextureRegion of the ImageLayer
	 * @return boolean */
	private boolean checkTransparencySupport (TextureRegion region) {	
		 Pixmap.Format format = region.getTexture().getTextureData().getFormat();	
		 return format != null && formatHasAlpha(format);
	}

	 //Check if pixel format supports alpha channel
	 private boolean formatHasAlpha(Pixmap.Format format) {
		  switch (format) {
		  case Alpha:
		  case LuminanceAlpha:
		  case RGBA4444:
		  case RGBA8888:
				return true;
		  default:
				return false;
		  }
	 }
```

I think this would cover any and all cases where an image could possible be of a format containing alpha values we support.

From my testing I've made sure it works in parity visually with Tiled using JPGs, PNGs, PNGs that don't have alpha support(RGB888 format), GIFs, and Greyscale PNGs that support transparency (LuminanceAlpha).

Reply:
Nice! I wonder if after this PR we could look at adding some `hasAlpha()` to Pixmap.Format, since this probably has uses elsewhere.